projects/components/src/multi-select/multi-select.component.ts

```diff
@@ -209,9 +213,11 @@ export class MultiSelectComponent<V> implements ControlValueAccessor, AfterConte
 
   public ngAfterContentInit(): void {
     this.allOptions$ = this.allOptionsList !== undefined ? queryListAndChanges$(this.allOptionsList) : EMPTY;
-    this.filteredOptions$ = combineLatest([this.allOptions$, this.searchSubject]).pipe(
+    this.filteredOptions$ = combineLatest([this.allOptions$, this.caseInsensitiveSearchSubject]).pipe(
       map(([options, searchText]) =>
-        options.filter(option => option.label.toLowerCase().includes(searchText.toLowerCase()))
+        isEmpty(searchText)
+          ? options.map(option => option)
```

Comment:
we don't need the map. just `options` will do

Reply:
no. Options is of type QueryList. Some weird angular thing. When we iterate over it then we get the underlying type. Map is required.